strr-platform-web/.env.example

```diff
@@ -19,6 +19,7 @@ NUXT_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
 NUXT_PAYMENT_PORTAL_URL="https://dev.account.bcregistry.gov.bc.ca/makepayment/"
 NUXT_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
 NUXT_BASE_URL="http://localhost:3000/" # app base url
+NUXT_HOUSING_STRR_URL="" # TODO: add housing strr base url here for redirects
```

Comment:
Did you want to add that to https://github.com/bcgov/STRR/blob/main/strr-platform-web/devops/vaults.env
or I can in another PR?